.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: Continuous Integration
+on:
+  push:
+
+jobs:
+  run-linters:
+    name: Run linters
```

Comment:
Everything is in one job called run-linters.

Reply:
The action runs them in sepperate jobs. So in total 3 different jobs will be instanced during this